ale/util.py

```diff
@@ -132,31 +132,54 @@ def read_pref(path):
                 pvlprefs = pvl.loads(preftext)
         except Exception as e:
             raise Exception(f'Failed to load IsisPreferences file [{path}]: {e}')
-            
+
         return pvlprefs
 
     argprefs = {}
     if isis_preferences:
         if isinstance(isis_preferences, dict):
             argprefs = isis_preferences
         elif os.path.isfile(isis_preferences):
-            argprefs = reaf_pref(isis_preferences)
+            argprefs = read_pref(isis_preferences)
         else:
             warnings.warn(f'{isis_preferences} does not exist, ignoring')
 
     try:
         homeprefs = read_pref(os.path.join(os.path.expanduser("~"), '.Isis', 'IsisPreferences'))
-    except: 
+    except Exception as e:
+        print(f'error: {e}')
```

Comment:
Saying the error happened while trying to read the User's preferences would be helpful.

Reply:
that was a debug print, probably just remove for now.

Reply:
Updated so it naturally fail if file exists and an exception is thrown.